### PR TITLE
Fix to notifications on organizations are sent only to one GitHub user

### PR DESCRIPTION
**Pull request by felixminom**
Hello. I'm raising this PR related to https://github.com/commercialhaskell/stackage/issues/5594.

Happy that we can make this small contribution, let us know if something is missing or could be improved.

### CODE REVIEW

File: test/Curator/SnapshotSpec.hs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import qualified Data.Set as S
+import Data.Text (unpack)
+import Distribution.Types.PackageName (mkPackageName)
+import Distribution.Types.VersionRange (noVersion)
+import Test.Hspec
+
+spec :: Spec
+spec = do
+  describe "pkgBoundsError" $
+    context "when there is more than one github pin" $
```
Comment (snoyberg) [inline test/Curator/SnapshotSpec.hs:15]: Was this supposed to say "ping" instead of "pin"?

**Review by snoyberg — COMMENTED** [review]
Thank you, it looks good to me, just a minor comment

File: test/Curator/SnapshotSpec.hs (hunk 1 above)
Comment (felixminom) [reply test/Curator/SnapshotSpec.hs:15]: Not really. Actually, I'm referring to github usernames that could be in the list of maintainers.

Comment (felixminom) [reply test/Curator/SnapshotSpec.hs:15]: I can change that, no problem. Sorry just realized that was a typo.

**Comment by bergmark** [comment]
Thanks @felixminom!

**Review by snoyberg — APPROVED** [review]

**Comment by snoyberg** [comment]
Thanks!

**Comment by felixminom** [comment]
Happy to help.

**Pull request merged**